tests/functional/pod_and_daemons/test_standalone_noobaa_pod_node_affinity.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.ocs.node import (
+    apply_node_affinity_for_noobaa_pod,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+class TestNoobaaPodNodeAffinity:
+    @pytest.fixture(scope="session", autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            """
+            Finalizer will take care of below activities:
+            1. Removes nodeaffinity to bring storage cluster with default values.
+
+            """
+            resource_name = constants.DEFAULT_CLUSTERNAME
+            if config.DEPLOYMENT["external_mode"]:
+                resource_name = constants.DEFAULT_CLUSTERNAME_EXTERNAL_MODE
```

Comment:
this is a standalone noobaa specific bug, do we really need this external mode check?